### PR TITLE
[ticket/15545] Fix no vertical space between format buttons

### PR DESCRIPTION
**Pull request by hubaishan**
Fix no vertical space between format buttons

PHPBB3-15545

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB3-15545

### CODE REVIEW

**Comment by hubaishan** [comment]
!set Styles

File: phpBB/styles/prosilver/theme/forms.css [hunk 1]
```diff
@@ -248,6 +248,10 @@ fieldset.submit-buttons input {
 	margin: 15px 0 2px 0;
 }
 
+.format-buttons .button {
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/forms.css]: this belongs in buttons.css also consider doing the same to `.form-buttons select` in this file...

Comment (hanakin) [reply phpBB/styles/prosilver/theme/forms.css]: also, this needs to be rebase against 3.3.

File: phpBB/styles/prosilver/theme/buttons.css [hunk 2]
```diff
@@ -152,6 +152,16 @@
 	max-width: 40%;
 }
 
+/* Format buttons
+--------------------------------------------- */
+.format-buttons .button {
+	margin-bottom: 3px;
+}
+
+.format-buttons input, .format-buttons select {
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/buttons.css]: I think you misunderstood me this should remain where it was but should also have the margin bottom added.

**Pull request merged**